src/app/sw.ts

```
@@ -24,3 +24,38 @@ const serwist = new Serwist({
+      await caches.open(RESET_TAG);
+
+      // Take control of existing pages, then reload them through the fresh worker.
+      await self.clients.claim();
+      const windows = await self.clients.matchAll({
+        includeUncontrolled: true,
+        type: 'window',
+      });
+      await Promise.all(
+        windows.map((client) =>
+          'navigate' in client
+            ? (client as WindowClient).navigate(client.url).catch(() => {})
```

ESLint

[error] 'WindowClient' is not defined.

(no-undef)